### PR TITLE
feat(vps): add main information tile

### PR DESCRIPTION
**Pull request by JeremyDec**
ref: MANAGER-4441
### 🚀 Feature
VPS new service information tile.
PR must not be merged before #2344

Signed-off-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>

### CODE REVIEW

File: packages/manager/modules/vps/src/dashboard/tile/information/information.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<oui-tile
```
Comment (JeremyDec) [inline packages/manager/modules/vps/src/dashboard/tile/information/information.html]: Changes concerns : 
- replacement of `<i>`by `<span>`
- some `::`
- remove of extra `data-`

File: packages/manager/modules/vps/src/dashboard/tile/information/information.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    <oui-tile-definition
+        term="{{ ::('vps_zone' | translate)+' ('+('vps_cluster' | translate)+')' }}"
+        description="{{ ::$ctrl.vps.zone+' ('+$ctrl.vps.cluster+')' }}"
+    >
+    </oui-tile-definition>
+    <oui-tile-definition
+        term="{{ ::'vps_dashboard_localisation' | translate }}"
+    >
+        <oui-tile-description>
+            <div data-ng-if="!$ctrl.hasMultipleRegions()">
```
Comment (frenautvh) [inline packages/manager/modules/vps/src/dashboard/tile/information/information.html]: It looks like you are using an array to store the single region. Can't you handle both case (single or multiple regions) with the same single ng-repeat ?

File: packages/manager/modules/vps/src/dashboard/tile/information/information.html (hunk 1 above)
Comment (Jisay) [reply packages/manager/modules/vps/src/dashboard/tile/information/information.html]: Simple question: I wonder why we remove the extra `data-`?

Comment (JeremyDec) [reply packages/manager/modules/vps/src/dashboard/tile/information/information.html]: Hmm they concerns ovh-ui-kit components attributes, so we don't need it. What do you think ?

File: packages/manager/modules/vps/src/dashboard/tile/information/information.html (hunk 2 above)
Comment (JeremyDec) [reply packages/manager/modules/vps/src/dashboard/tile/information/information.html]: Yes in fact, and also, `detailedRegions` and `regionsGroup` are the same. Removing `detailedRegions`.

File: packages/manager/modules/vps/src/dashboard/tile/information/information.html (hunk 1 above)
Comment (Jisay) [reply packages/manager/modules/vps/src/dashboard/tile/information/information.html]: It's optional indeed. 
I add `data-` to non standard html attribute in order to recognize them more easily. But this is my point of view :smile:

File: packages/manager/modules/vps/src/dashboard/tile/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import angular from 'angular';
+
+import informationTile from './information';
+
+const moduleName = 'ovhManagerVpsDashboardTile';
+
+angular.module(moduleName, [informationTile]);
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/tile/index.js]: No translations, no ui-kit, no ui-router ?

File: packages/manager/modules/vps/src/dashboard/tile/information/index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import 'angular-translate';
+import ngOvhTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+
+import component from './information.component';
+
+const moduleName = 'ovhManagerVpsDashboardTileInformation';
+
+angular
+  .module(moduleName, [ngOvhTranslateAsyncLoader, 'pascalprecht.translate'])
+  .component('ovhManagerVpsDashboardTileInformation', component)
```
Comment (FredericEspiau) [inline packages/manager/modules/vps/src/dashboard/tile/information/index.js]:
```suggestion
  .component('vpsDashboardTileInformation', component)
```

File: packages/manager/modules/vps/src/dashboard/tile/index.js (hunk 3 above)
Comment (JeremyDec) [reply packages/manager/modules/vps/src/dashboard/tile/index.js]: No ui-router, as there is no routing set here.
No translations, as the child have it for itself.
Ok for ui-kit :)

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10632.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10632.0 ✘  
* 
  * Build ✘

**Pull request closed without merging**